OpenRA.Mods.Common/AudioLoaders/OggLoader.cs

```diff
@@ -106,12 +104,10 @@ public override int Read(byte[] buffer, int offset, int count)
 				count -= count % format.reader.Channels;
 
 				// Get the buffer, creating a new one if none exists or the existing one is too small.
```

Comment:
this comment has become outdated

Reply:
The comment is still accurate, it's what the new helper method is doing.

Reply:
I meant that it doesn't belong here, if anything it should be in the function, but even then it's self-evident when looking at the code. Not to mention the new function name

Reply:
I'm glad it's clear. Comment removed.